docs/manual/scala/guide/services/ServiceClients.md

```diff
@@ -76,14 +76,9 @@ With the above "hello" example we could adjust the configuration by defining pro
 
 ### Circuit breaker metrics
 
-Lagom allows you to publish metrics for circuit breakers via a metrics service. To enable this service, mix in the [`MetricsServiceComponents`](api/com/lightbend/lagom/scaladsl/server/status/MetricsServiceComponents.html) trait into your application, and add the provided `metricsServiceBinding` to your service bindings in your `lagomServer` declaration, like so:
-
-@[metrics-service](code/ServiceClients.scala)
-
-The service provides the following endpoints:
+Lagom allows you to publish metrics for circuit breakers via a metrics service. A `MetricsService` is bound under the covers in every Lagom server. Then, for each instance of your service you can request the following endpoints:
```

Comment:
I should probably undo this changes in the docs. (not doing now because it's late night)